src/components/CocktailCard.jsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import supabase from '../config/supabaseClient'
+
+const CocktailCard = ({ cocktail, onDelete }) => {
+    
+    const handleDelete = async() => {
```

Comment:
Two things here:

1. While it is good that you're passing the `onDelete` function here, the logic should be handled on the parent component. It is preferred that the parent components handle all the logic and the child components react to data changes or emit events to let the parent know something changed. The `handleDelete` on [line 33](https://github.com/Victorloma/cocktails/pull/1/files#diff-47130132c55edcceca19b447c10b8815b9892df13f58494c6681dba887522900R33)  should call `onDelete` with the respective `id` and let the parent deal with that action.
2. Normally you don't want to have data manipulation inside a component (nor here or the parent). You want to have access to functions that let you do data manipulation but not do it directly from the component. What if you'd want to delete a cocktail from a different component? You'd have to implement the same logic in a different one. This logic should go into its own function in a service.

For example:

`src/services/cocktails.service.js`

```
export const deleteCocktail = async (cocktailId) => await supabase
  .from('cocktails')
  .delete()
  .eq('id', cocktail.id)
  .select();
```

Reply:
This took me forever, but I got it to work :p
Not sure if I did it right though..